lib/errata/errata_requests.py

```diff
@@ -166,7 +178,26 @@ def format_advisory_data(advisory_data):
     final_response["advisory_details"] = advisory_details
 
     total_bugs = 0
-
+    jira_issues_details = []
+    total_jira_issues = 0
+    jira_issue_summary = dict()
+    if jira_issues_data:
+        for jira_issue in jira_issues_data:
+            jira_issue_detail = {
+                "id_jira": jira_issue.get("id_jira"),
+                "key": jira_issue.get("key"),
+                "summary": jira_issue.get("summary"),
+                "status": jira_issue.get("status"), 
+                "is_private": jira_issue.get("is_private"),
+                "labels": jira_issue.get("labels"),
+            }
+            jira_issues_details.append(jira_issue_detail)
+            total_jira_issues += 1
+
+            # Counting the JIRA issue statuses
+            if jira_issue_detail["status"] not in jira_issue_summary:
+                jira_issue_summary[jira_issue_detail["status"]] = 0
+            jira_issue_summary[jira_issue_detail["status"]] += 1
```

Comment:
Can you explain what you're trying to achieve here? I'm not exactly clear what the difference is between `jira_issue_summary` and `jira_issue_detail`

Reply:
**jira_issue_detail["status"]:**
This captures the status of the current JIRA issue we're iterating over.

**if jira_issue_detail["status"] not in jira_issue_summary::**
This checks if the status of the current JIRA issue exists in our jira_issue_summary dictionary. If it doesn't, we initialize a counter for this status in the next line.

**jira_issue_summary[jira_issue_detail["status"]] = 0:**
We're initializing the count of this particular status to zero. This is a setup for our tally of different JIRA issue statuses.

**jira_issue_summary[jira_issue_detail["status"]] += 1:**
This line increments the count of the current status by one, effectively counting the number of occurrences of each status.

**To differentiate between jira_issue_detail and jira_issue_summary:**
jira_issue_detail: Contains specific details about a single JIRA issue, including its status.
jira_issue_summary: Is a dictionary that acts as a counter, keeping track of the number of JIRA issues that have each specific status.

So in short, I'm tallying the number of JIRA issues per status in jira_issue_summary using details from each jira_issue_detail.